packages/openbox.rb

```diff
@@ -33,25 +34,87 @@ def self.build                   # the steps required to build the package
+    #while ! xdpyinfo -display ${DISPLAY} >/dev/null 2>&1; do
+    #  echo -n \".\"
+    #  sleep 0.50s
+    #  export DISPLAY=100.115.92.2:0
+    #  COUNT=$(( COUNT + 1 ))
+    #  if [ \"${COUNT}\" -ge \"${MAX_ATTEMPTS}\" ]; then
+    #    echo \"  Gave up waiting for X server on ${DISPLAY}\"
+    #    exit 1
+    #  fi
+    #done
+    #echo \"  Done - X server is ready!\"
+    #EOF"
     system "echo 'sleep 7' >> startopenbox"
     system "echo 'export DISPLAY=100.115.92.2:0' >> startopenbox"
-    system "echo 'openbox' >> startopenbox"
-    system "echo 'echo XServer closed or logged out session' >> startopenbox"
+    system "echo 'openbox \"$@\"' >> startopenbox"
     system "echo 'startsommelier' >> startopenbox"
-    system "echo 'export DISPLAY=:0' >> startopenbox"
-    system "install -Dm755 startopenbox #{CREW_DEST_PREFIX}/bin/startopenbox"
+    # For Chrome OS/Chromium OS/Cloudready without android arc container support
+    # Use 'xinit'
+    system "if [[ cat /etc/lsb-release | grep ARC != "" ]]; then echo 'Your OS have Android Support :)'; else if [[ whereis xinit | grep /usr/local/bin/xinit != /usr/local/bin/xinit ]]; then echo 'To use this package, You need to install `xinit` package'; else echo 'pkill twm && openbox &' >> /usr/local/etc/X11/xinit/xinitrc; fi"
+    ## For i686 user
+    # system "echo 'The unstable method, feel free to test it:)'"
+    ## Download ARChon Runtime for Chrome
+    # puts "Downloading ARChon Android Runtime for Chrome...".lightblue
+    # system "wget http://archon.vf.io/ARChon-v2.1.0-x86_32.zip"
+    ## Download XServer Extension for ARChon Runtime
+    # puts "Downloading XServer Extension for Chrome...".lightblue
+    # system "wget https://github.com/supechicken666/X-Server-Extension-for-Chrome/raw/main/au.com.darkside.XServer.zip"
+    # system "echo '#!/bin/bash' > startopenbox_arc-extension"
+    # system "cat <EOF>> startopenbox_arc-extension
+    ## Wait for X server extension
+    # MAX_ATTEMPTS=120 # About 60 seconds
+    # COUNT=0
+    # echo Please open `au.com.darkside.XServer` extension from app drawer
+    # stopsommelier
+    # echo -n \"Waiting for X server to be ready...\"
+    # while ! xdpyinfo -display ${DISPLAY} >/dev/null 2>&1; do
+    #  echo -n \".\"
+    #  sleep 0.50s
+    #  export DISPLAY=127.0.0.1:0
+    #  COUNT=$(( COUNT + 1 ))
+    #  if [ \"${COUNT}\" -ge \"${MAX_ATTEMPTS}\" ]; then
+    #    echo \"  Gave up waiting for X server on ${DISPLAY}\"
+    #    exit 1
+    #  fi
+    #done
+    #echo \"  Done - X server is ready!\"
+    #xclock &
+    #xterm &
+    #openbox &
+    #EOF"
+    # system "unzip *.zip && mv */ ~/Downloads"
+    # puts "Install the two unpack extension to chrome with entension developer mode on".lightblue  
+    # system "if [[ $(uname -m) = i686 ]]; then 'install -Dm755 startopenbox_arc-extension #{CREW_DEST_PREFIX}/bin/startopenbox'; else 'install -Dm755 startopenbox #{CREW_DEST_PREFIX}/bin/startopenbox'; fi"
```

Comment:
No need to include all the commented lines.  Please remove.

Reply:
Done